pkg/users/index.html

```diff
@@ -123,7 +123,15 @@ <h1 translatable="yes">Account not available or cannot be edited.</h1>
     <div class="panel panel-default" id="account-authorized-keys">
       <div class="panel-heading">
         <div class="pull-right">
-          <button data-toggle="modal" data-target="#add-authorized-key-dialog" data-original-title="Add Public Key" class="btn btn-primary accounts-self-privileged fa fa-plus" id="authorized-key-add" title=""></button>
+          <button data-toggle="modal"
+                  data-target="#add-authorized-key-dialog"
+                  data-original-title="Add Public Key"
+                  data-placement="left"
+                  class="btn btn-primary accounts-self-privileged fa fa-plus"
+                  id="authorized-key-add"
+                  title=""
+                  type="button">
```

Comment:
That raised an eyebrow, but looks correct according to https://developer.mozilla.org/de/docs/Web/HTML/Element/button .